core/zoe/docs/SWAPS.md

```diff
@@ -0,0 +1,78 @@
+# Swaps
+
+If I want to trade one kind of asset for another kind, I could send
+you the asset and ask you to send me the other kind back. But, you
+could choose to behave opportunistically: receive my asset and give
+nothing back. To solve this problem, swap contracts allow users to
+securely trade one kind of eright for another kind, leveraging Zoe
+for escrow and offer-safety. At no time does any user have the
+opportunity to behave opportunistically.
```

Comment:
Hmm, I've been taught that repeating the same word ("opportunistically") can often be desirable in explanatory writing because using a slightly different word makes the user wonder if there's a substantial difference between "cheating" and "acting opportunistically." I like the change to `ability` though. I'll make that change. Would it be possible to separate copy-editing from factual errors and do a later run through all the documentation for things that aren't factual errors but may be improved by rewording?